samples/Xkcd/sort.ps1

```diff
@@ -0,0 +1,2 @@
+yml-sorter --input CodeModel-xkcd.yaml
```

Comment:
Did you intend to commit this script?

Reply:
It was used to test comparing code model files prior to writing the order numbering algorithm. But, it didn't work that well anyway. So, I'll likely delete it.